src/tetris/ui/dialog/evaluationView.ts

```diff
@@ -0,0 +1,192 @@
+import Profile from "tetris/profiler/profile";
+import {Timeline, DataSet} from "vis";
+import Game from "tetris/game";
+import BiasEvaluation from "tetris/biasEngine/biasEvaluation";
+import Profiler from "tetris/profiler/profiler";
+import BiasEventType from "tetris/biasEngine/biasEventType";
+import BiasEngine from "tetris/biasEngine/biasEngine";
+
+const CSS_CLASS_DATA_WRAPPER = "evaluation-data-wrapper";
+const BASE64_IMAGE_PREFIX = "data:image/png;base64, ";
+const VIEW_TITLE_WRAPPER_CLASS = "view-title";
+
+const TIMELINE_GROUP_BIAS_EVENTS = "Bias Events";
+const TIMELINE_GROUP_BIAS_VALUE = "Bias Value";
+
+export default class EvaluationView {
+
+	//region public members
+	//endregion
+
+	//region publsic methods
+	public show(): EvaluationView {
+		this._htmlElement.classList.add('visible');
+		this._update();
+		return this;
+	}
+	//endregion
+
+	//region constructor
+	public constructor(game: Game) {
+		this._htmlElement = document.getElementById("evaluation-view");
+		this._informationContainer = this._htmlElement.querySelector("#evaluation-view-basic-information");
+		this._pictureContainer = this._htmlElement.querySelector("#evaluation-view-picture");
+		this._timelineContainer = this._htmlElement.querySelector("#evaluation-view-timeline");
+		this._game = game;
+		this._displayTitle("Evaluate your player profile");
+	}
+	//endregion
+
+	//region private members
+	private _game: Game;
+	protected readonly _htmlElement: HTMLElement;
+	private readonly _informationContainer: HTMLDivElement;
+	private readonly _pictureContainer: HTMLImageElement;
+	private readonly _timelineContainer: HTMLDivElement;
+	private _timeline: Timeline;
+
+	private get profiler(): Profiler {
+		return this._game.profiler;
+	}
+
+	private get profile(): Profile {
+		return this.profiler.profile;
+	}
+
+	private get biasEvaluation(): BiasEvaluation {
+		return this._game.biasEngine.biasEvaluation;
+	}
+
+	private get biasEngine(): BiasEngine {
+		return this._game.biasEngine;
+	}
+	//endregion
+
+	//region private methods
+	private _displayBasicInformation(): void {
+		const nodes = [];
+		nodes.push(EvaluationView._createDataWrapper('age', this.profile.age));
```

Comment:
Refactor using Object.keys() as discussed.

Reply:
Discussed it again and decided: no!